igvc_navigation/launch/follower.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<launch>
+    <!-- Follower -->
+    <node name="path_follower" pkg="igvc_navigation" type="path_follower" output="screen" >
+        <!-- Maxmimum wheel velocity allowed before stopping motors -->
+        <param name="maximum_vel" type="double" value="2"/>
+        <param name="axle_length" type="double" value="0.52"/>
+        <!-- Distance along the path to generate smooth control law for  -->
+        <param name="lookahead_dist" type="double" value="0.4"/>
+        <!-- Frequency (hz) at which simulation is done -->
+        <param name="simulation_frequency" type="double" value="10.0"/>
+        <!-- How many seconds in the future should be simulated-->
+        <param name="simulation_horizon" type="double" value="5.0"/>
+        <param name="target_reached_distance" type="double" value="0.1"/>
```

Comment:
Also, add short descriptions that describe these parameters (`target_reached_distance` and `target_move_threshold`)